classes/passwd.oeclass

```diff
@@ -19,18 +19,14 @@ GROUP_FILES ?= "${SRCDIR}/group"
 PASSWD_POSTFUNC:machine = " do_install_passwd"
 do_install[postfuncs] += "${PASSWD_POSTFUNC}"
 do_install_passwd () {
-	i=0
 	for f in ${PASSWD_FILES} ; do
-		mkdir -p ${D}/${passwddir}
-		let i=$i+1
-		cp $f ${D}/${passwddir}/${PN}.$i
+		mkdir -p ${D}/${passwddir}/${PN}
+		cp $f ${D}/${passwddir}/${PN}/
```

Comment:
Well, if one is taking files from $B one might not have that much control over their basenames, so it would require somewhat more than a rename. But no-one does that right now, so I'll drop the complexity and just add -n, and maybe a link to this very PR so we can revisit should the need arise.